regression/cbmc-incr-smt2/dynamic-memory/assert_dynamic.c

```diff
@@ -0,0 +1,17 @@
+#include <assert.h>
```

Comment:
⛏️ As we have a test checking that the object is not dynamic, should we avoid the nondeterministic boolean so that one verification leads to `FAILURE` and one to `SUCCESS`?
Otherwise we never check for `SUCCESS` (`UNSAT` on the SMT level).